docker/docker-compose.yml

```diff
@@ -67,30 +72,34 @@ services:
     networks:
       - nightfall_network
     environment:
+      ALWAYS_DO_TRUSTED_SETUP: ${ALWAYS_DO_TRUSTED_SETUP}
+      BLOCKCHAIN_URL: ${BLOCKCHAIN_URL}
+      BLOCKCHAIN_WS_HOST: ${BLOCKCHAIN_WS_HOST:-blockchain}
+      BLOCKCHAIN_PORT: ${BLOCKCHAIN_PORT:-8546}
```

Comment:
`BLOCKCHAIN_PORT` on top of `BLOCKCHAIN_URL` (?)
`DAI_RESTRICT` should be on top of `DEPLOY_`

For me obvs this is nice but from experience alphabetical order is also hard to maintain...
What do you think? Could a comment help perhaps??

Reply:
Completely missed the "first letter" thing :upside_down_face: